### PR TITLE
Deadcode [wip]

### PR DESCRIPTION
**Pull request by mmetc**
Some low-level refactoring. All interventions are local, should not change behavior and do not cross function boundaries.

### CODE REVIEW

File: pkg/types/ip.go [hunk 1]
```diff
@@ -38,17 +38,19 @@ func Addr2Ints(any string) (int, int64, int64, int64, int64, error) {
...
-		if ip == nil {
-			return -1, 0, 0, 0, 0, fmt.Errorf("invalid address")
-		}
-		sz, start, end, err := IP2Ints(ip)
-		if err != nil {
-			return -1, 0, 0, 0, 0, errors.Wrapf(err, "while parsing ip %s", any)
-		}
-		return sz, start, end, start, end, nil
 	}
+
```
Comment (buixor) [inline pkg/types/ip.go:42]: not 100% on you with this one for comprehension but I won't push :)

**Review by buixor — APPROVED** [review]
overall lgtm :)

File: pkg/csconfig/simulation.go [hunk 2]
```diff
@@ -42,10 +42,9 @@ func (c *Config) LoadSimulation() error {
 	rcfg, err := ioutil.ReadFile(c.ConfigPaths.SimulationFilePath)
 	if err != nil {
 		return errors.Wrapf(err, "while reading '%s'", c.ConfigPaths.SimulationFilePath)
-	} else {
-		if err := yaml.UnmarshalStrict(rcfg, &simCfg); err != nil {
-			return fmt.Errorf("while unmarshaling simulation file '%s' : %s", c.ConfigPaths.SimulationFilePath, err)
-		}
+	}
+	if err := yaml.UnmarshalStrict(rcfg, &simCfg); err != nil {
+		return fmt.Errorf("while unmarshaling simulation file '%s' : %s", c.ConfigPaths.SimulationFilePath, err)
```
[the remark this answers is not in the document]
Comment (mmetc) [reply pkg/csconfig/simulation.go:47]: Well, the Printf stuff is in the other bag of commits..

File: pkg/types/ip.go (hunk 1 above)
Comment (mmetc) [reply pkg/types/ip.go:42]: I just liberated the else there, changed nothing more

**Pull request merged**